be/src/exec/workgroup/work_group.h

```diff
@@ -59,13 +59,18 @@ class WorkGroup {
 
     int64_t vruntime_ns() const { return _vruntime_ns; }
     int64_t real_runtime_ns() const { return _vruntime_ns * _cpu_limit; }
+
+    int64_t growth_vruntime_ns() const { return _vruntime_ns * _cpu_limit - _last_vruntime_ns; }
+    void update_last_vruntime_ns(int64_t last_vruntime_ns) { _last_vruntime_ns = last_vruntime_ns; }
```

Comment:
Maybe they should be renamed to `growth_real_runtime_ns` and `update_last_real_runtime_ns`.